### PR TITLE
rubocop new clean version

### PR DESCRIPTION
**Pull request by franciscoabalan**
## Summary
Fix old branches and push a new rubocop version from kickoff-api branch

## Trellocard
https://trello.com/c/uWKkmskS/8-configurar-rubocop

## Screenshots
Captura de Pantalla 2019-10-01 a la(s) 13 06 03

### CODE REVIEW

File: Gemfile
Comment (aleochoam) [inline Gemfile]: In Wolox, we don't use this line, instead you could add the following line into the  rubocop.yml to avoid that warning:

```
Style/FrozenStringLiteralComment:
  Enabled: false

```

That way, you don't have to include  that line in ALL the files

Comment (aleochoam) [inline Gemfile]: Let's delete this line and we are good to go!

File: Gemfile
Comment (aleochoam) [inline Gemfile]: We usually lock the gem version to keep the dependencies under control, for example [this](https://github.com/wolox-training/pj-fa-rails/pull/4/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R50) gem

File: Gemfile
Comment (franciscoabalan) [reply Gemfile]: Done

Comment (franciscoabalan) [reply Gemfile]: Done

File: Gemfile
Comment (franciscoabalan) [reply Gemfile]: Done

Comment (joredudiaz) [inline Gemfile]: please put this line at `group :development, :test`, we use this gem just for development and test environment

Comment (franciscoabalan) [reply Gemfile]: Done.. only missing push it

**Review by joredudiaz — APPROVED** [review]
let's wait for another approval 👍

**Review by aleochoam — APPROVED** [review]

**Pull request merged**
